---
layout: archive
title: ""
permalink: /AI_for_Biochemistry/
author_profile: true
---

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(241, 241, 241);">
</br>
<h2>AI for Biochemistry</h2>
</br>
<p style="text-align:justify">
  Artificial intelligence (AI) is revolutionizing biochemistry in omics, protein, and molecular research. 
  These advancements hold the potential to accelerate drug discovery and improve our understanding of complex biochemical processes. 
  Traditional methods for transcriptomics analysis, protein representation learning, and molecule modeling are time-consuming and often limited by the complexity of biological systems. 
  The vast amount of data and intricate biochemical interactions pose significant challenges, leading to lengthy development cycles and high costs. 
  In biochemistry, datasets remain limited, with significant room for exploration, such as cancer-specific gene expression analysis, and the rich domain knowledge in science has not been fully exploited. 
  While large models are beginning to show promise across research areas, they also present new demands for model inference and deployment under constrained conditions.  
</p>
<p style="text-align:justify">
  Our Ideaï¼š
  <ul>
    <li>Data-Level Improvements: We propose building new databases and toolkits for pattern recognition and incorporating domain knowledge into datasets.</li>
    <li>Model-Level Optimization: We aim to develop new architectures, efficient fine-tuning methods, and structural knowledge priors to optimize models and training strategies.</li>
    <li>Application-Level Enhancements: Our approach includes creating new biochemistry platforms and utilizing model compression and parallel training to boost performance and scalability.</li>
  </ul>
</p>
</br>
  
<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>Multimodal Artificial Intelligence Large Models for Small-Molecule Drug Generation and Description</h3>
<p>Paper Link: <a href="https://arxiv.org/abs/2308.06911">https://arxiv.org/abs/2308.06911</a></p>
<img style="width: 100%;padding: 3%;" src="/images/GIT-Mol.png" />
<p style="text-align:justify">
  We are dedicated to the development of an innovative multimodal large model in the fields of drug discovery and molecular biology, aiming to provide new perspectives and solutions for understanding and utilizing multimodal molecular data. 
  To this end, we have designed and implemented a multimodal AI large model capable of simultaneously processing images, graphs, and text. 
  Through a novel cross-modal fusion approach (identifying and extracting correlations between different modalities and implementing cross-fusion of information, thereby generating a unified embedding that integrates multimodal information), our model achieves an organic interaction and efficient fusion among these three modalities. 
  Our model has demonstrated outstanding performance across various small-molecule drug tasks, with the ability to understand and handle multimodal molecular data. 
  It has accomplished accurate prediction of molecular properties, as well as the generation and description of molecules in response to textual prompts, showing superior performance in comparison with models of similar levels.
</p>
</br>
</div>

</br>

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>Deep Neural Network-Based Research on Small-Molecule Drugs</h3>
<p>Paper Link: <a href="https://www.biorxiv.org/content/10.1101/2023.08.15.553467v1.abstract">https://www.biorxiv.org/content/10.1101/2023.08.15.553467v1.abstract</a></p>
<img style="width: 100%;padding: 3%;" src="/images/3D-Mol.png" />
<p style="text-align:justify">
  By designing novel molecular representation methods and utilizing the force field computations and knowledge-base optimization features of RDKit, we generate 3D conformations of molecules from molecular topological graphs. 
  Subsequently, we decompose the molecular conformations into point-edge graphs, bond-angle graphs, and dihedral-angle graphs to comprehensively characterize the spatial information of molecules and extract the 3D structure of small-molecule drugs. 
  We have designed a molecular encoder based on a message-passing model and pre-trained their model using extensive unlabeled small-molecule data through a self-supervised contrastive learning approach (using molecular topological graphs). 
  Our AI model has achieved state-of-the-art (SOTA) performance in several aspects of small-molecule drug property prediction.
</p>
<a></a>
</br>
</div>

</br>

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>A Self-feedback Knowledge Elicitation Approach for Chemical Reaction Predictions</h3>
<p>Paper Link: <a href="https://arxiv.org/abs/2404.09606">https://arxiv.org/abs/2404.09606</a></p>
<img style="width: 100%;padding: 3%;" src="/images/CRP.png" />
<p style="text-align:justify">
  The task of chemical reaction predictions (CRPs) plays a pivotal role in advancing drug discovery and material science. 
  However, its effectiveness is constrained by the vast and uncertain chemical reaction space and challenges in capturing reaction selectivity, particularly due to existing methods' limitations in exploiting the data's inherent knowledge. 
  To address these challenges, we introduce a data-curated self-feedback knowledge elicitation approach. 
  This method starts from iterative optimization of molecular representations and facilitates the extraction of knowledge on chemical reaction types (RTs). 
  Then, we employ adaptive prompt learning to infuse the prior knowledge into the large language model (LLM). 
  As a result, we achieve significant enhancements: a 14.2% increase in retrosynthesis prediction accuracy, a 74.2% rise in reagent prediction accuracy, and an expansion in the model's capability for handling multi-task chemical reactions. 
  This research offers a novel paradigm for knowledge elicitation in scientific research and showcases the untapped potential of LLMs in CRPs.
</p>
</br>
</div>

</br>

</br>
</div>
