---
layout: archive
title: ""
permalink: /AI_for_Traditional_Chinese_Medicine/
author_profile: true
---

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(241, 241, 241);">
</br>
<h2>AI for Traditional Chinese Medicine</h2>
</br>
<p style="text-align:justify">
  Recent years, a wealth of studies have demonstrated that Traditional Chinese Medicine (TCM) and Western medicine share the same theoretical foundation at the molecular level. 
  However, current information on the chemical components and metabolism mechanisms of TCM remains insufficient, rendering the relationship between drug components and pharmacological effects unclear and thereby hindering targeted TCM diagnosis and treatment. 
  Consequently, the modernization of TCM urgently requires the establishment of a comprehensive and highly reliable TCM modernization database. 
  Although recent efforts have been made to construct TCM databases that incorporate information on herbs, ingredients, and targets to support related research, these databases are often plagued by varying degrees of information loss. 
  Specifically, existing databases tend to represent relationships in a binary manner, whereas biological processes are inherently complex and diverse. 
</p>
<p style="text-align:justify">
  Our Ideaï¼š
  <ul>
    <li>Database Construction: We create the currently largest Traditional Chinese Medicine Modernization database (TCMM) to comprehensively address the current information gap.</li>
    <li>Exploration and Discovery: Leveraging this extensive database, we explore three key areas: prescription generation, knowledge discovery, and bioactive constituent identification, and have discovered conclusions consistent with biological experimental results.</li>
    <li>Application-level enhancement: We develop a publicly accessible web interface to facilitate data browsing and online prescription generation, significantly enhancing the integration of TCM with contemporary medical practices.</li>
  </ul>
</p>
</br>

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>Identification of bioactive constituents for colitis from traditional Chinese medicine prescription via deep neural network</h3>
<p>Paper Link: <a href="https://www.biorxiv.org/content/10.1101/2023.05.31.542690.abstract">https://www.biorxiv.org/content/10.1101/2023.05.31.542690.abstract</a></p>
<img style="width: 100%;padding: 3%;" src="/images/TCM1.png" />
<p style="text-align:justify">
  Colitis is a commonly encountered inflammatory disease in colon tissue, which can be triggered by various causes. 
  Although traditional Chinese medicine (TCM) has been utilized for the treatment of colitis, it is still a great challenge to identify the major bioactive constituents and their modes of action among thousands of ingredients in TCM prescriptions. 
  Inspired by the success of artificial intelligence and deep learning methods, we proposed a deep neural network (DNN) for TCM prescription recommendation. 
  We constructed a graph-based DNN with 9,845 nodes and 161,950 edges, which integrated microscopic information including bioactive molecules, protein targets, and extracted features of prescriptions through feature embedding. 
  A novel and efficient data augmentation strategy was implemented to expand the sample size based on 378 collected TCM prescriptions. 
  Network pharmacology study revealed that the 10 most frequent ingredients in generated prescriptions were associated with multiple inflammatory signaling pathways. 
  To verify the bioactive constituents in the generated prescriptions, 5 selected constituents were administrated to BALB/c mice with colitis. 
  Suppressive effects of disease progression and proinflammatory factors comparable to sulfasalazin were observed with these compounds, revealing the effectiveness of our artificial intelligence strategy on idetification of bioactive constituents from TCM prescriptions.
</p>
</br>
</div>

</br>

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>RNA Splicing Prediction with Large Language model</h3>
<p>Paper Link: <a href="https://www.biorxiv.org/content/10.1101/2023.01.31.526427v2.abstract">https://www.biorxiv.org/content/10.1101/2023.01.31.526427v2.abstract</a></p>
<img style="width: 100%;padding: 3%;" src="/images/SpliceBERT.png" />
<p style="text-align:justify">RNA splicing is an important post-transcriptional process of gene expression in eukaryotic cells. Predicting RNA splicing from primary sequences can facilitate the interpretation of genomic variants. We developed a novel self-supervised pre-trained language model, SpliceBERT, to improve sequence-based RNA splicing prediction. Pre-training on pre-mRNA sequences from vertebrates enables SpliceBERT to capture evolutionary conservation information and characterize the unique property of splice sites. SpliceBERT also improves zero-shot prediction of variant effects on splicing by considering sequence context information, and achieves superior performance for predicting branchpoint in the human genome and splice sites across species. Our study highlighted the importance of pre-training genomic language models on a diverse range of species and suggested that pre-trained language models were promising for deciphering the sequence logic of RNA splicing.
</p>
</br>
</div>

</br>

</br>
</div>
