---
layout: archive
title: ""
permalink: /AI_for_Biochemistry/
author_profile: true
---

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(241, 241, 241);">
</br>
<h2>AI for Biochemistry</h2>
</br>
<p style="text-align:justify">
  Artificial intelligence (AI) is revolutionizing biochemistry in omics, protein, and molecular research. 
  These advancements hold the potential to accelerate drug discovery and improve our understanding of complex biochemical processes. 
  Traditional methods for transcriptomics analysis, protein representation learning, and molecule modeling are time-consuming and often limited by the complexity of biological systems. 
  The vast amount of data and intricate biochemical interactions pose significant challenges, leading to lengthy development cycles and high costs. 
  In biochemistry, datasets remain limited, with significant room for exploration, such as cancer-specific gene expression analysis, and the rich domain knowledge in science has not been fully exploited. 
  While large models are beginning to show promise across research areas, they also present new demands for model inference and deployment under constrained conditions.  
</p>
<p style="text-align:justify">
  Our Ideaï¼š
  <ul>
    <li>Data-Level Improvements: We propose building new databases and toolkits for pattern recognition and incorporating domain knowledge into datasets.</li>
    <li>Model-Level Optimization: We aim to develop new architectures, efficient fine-tuning methods, and structural knowledge priors to optimize models and training strategies.</li>
    <li>Application-Level Enhancements: Our approach includes creating new biochemistry platforms and utilizing model compression and parallel training to boost performance and scalability.</li>
  </ul>
</p>
</br>
  
<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>GIT-Mol: A Multi-modal Large Language Model for Molecular Science with Graph, Image, and Text</h3>
<p>Paper Link: <a href="https://arxiv.org/abs/2308.06911">https://arxiv.org/abs/2308.06911</a></p>
<img style="width: 100%;padding: 3%;" src="/images/GIT-Mol.png" />
<p style="text-align:justify">
  Large language models have made significant strides in natural language processing, enabling innovative applications in molecular science by processing textual representations of molecules. 
  However, most existing language models cannot capture the rich information with complex molecular structures or images. 
  In this paper, we introduce GIT-Mol, a multi-modal large language model that integrates the Graph, Image, and Text information. 
  To facilitate the integration of multi-modal molecular data, we propose GIT-Former, a novel architecture that is capable of aligning all modalities into a unified latent space. 
  We achieve a 5%-10% accuracy increase in properties prediction and a 20.2% boost in molecule generation validity compared to the baselines. 
  With the any-to-language molecular translation strategy, our model has the potential to perform more downstream tasks, such as compound name recognition and chemical reaction prediction.
</p>
</br>
</div>

</br>

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>3D-Mol: A Novel Contrastive Learning Framework for Molecular Property Prediction with 3D Information</h3>
<p>Paper Link: <a href="https://www.biorxiv.org/content/10.1101/2023.08.15.553467v1.abstract">https://www.biorxiv.org/content/10.1101/2023.08.15.553467v1.abstract</a></p>
<img style="width: 100%;padding: 3%;" src="/images/3D-Mol.png" />
<p style="text-align:justify">
  Molecular property prediction offers an effective and efficient approach for early screening and optimization of drug candidates. 
  Although deep learning based methods have made notable progress, most existing works still do not fully utilize 3D spatial information. 
  This can lead to a single molecular representation representing multiple actual molecules. 
  To address these issues, we propose a novel 3D structure-based molecular modeling method named 3D-Mol. 
  In order to accurately represent complete spatial structure, we design a novel encoder to extract 3D features by deconstructing the molecules into three geometric graphs. 
  In addition, we use 20M unlabeled data to pretrain our model by contrastive learning. 
  We consider conformations with the same topological structure as positive pairs and the opposites as negative pairs, while the weight is determined by the dissimilarity between the conformations. 
  We compare 3D-Mol with various state-of-the-art(SOTA) baselines on 7 benchmarks and demonstrate our outstanding performance in 5 benchmarks.
</p>
<a></a>
</br>
</div>

</br>

<div style="width: 100%;padding-left: 3%;padding-right: 3%;background-color: rgb(226, 226, 226);">
</br>
<h3>A Self-feedback Knowledge Elicitation Approach for Chemical Reaction Predictions</h3>
<p>Paper Link: <a href="https://arxiv.org/abs/2404.09606">https://arxiv.org/abs/2404.09606</a></p>
<img style="width: 100%;padding: 3%;" src="/images/CRP.png" />
<p style="text-align:justify">
  The task of chemical reaction predictions (CRPs) plays a pivotal role in advancing drug discovery and material science. 
  However, its effectiveness is constrained by the vast and uncertain chemical reaction space and challenges in capturing reaction selectivity, particularly due to existing methods' limitations in exploiting the data's inherent knowledge. 
  To address these challenges, we introduce a data-curated self-feedback knowledge elicitation approach. 
  This method starts from iterative optimization of molecular representations and facilitates the extraction of knowledge on chemical reaction types (RTs). 
  Then, we employ adaptive prompt learning to infuse the prior knowledge into the large language model (LLM). 
  As a result, we achieve significant enhancements: a 14.2% increase in retrosynthesis prediction accuracy, a 74.2% rise in reagent prediction accuracy, and an expansion in the model's capability for handling multi-task chemical reactions. 
  This research offers a novel paradigm for knowledge elicitation in scientific research and showcases the untapped potential of LLMs in CRPs.
</p>
</br>
</div>

</br>

</br>
</div>
